app/routes/_dashboard.podcasts.tsx

```diff
@@ -0,0 +1,43 @@
+import { json } from "@remix-run/node";
+import { useLoaderData, useRouteError } from "@remix-run/react";
+import { getPodcasts } from "~/models/content.server";
+import CardWithMultipleContent from "~/components/card/CardWithMultipleContent";
+import ErrorComponent from "~/components/Error";
+
+export const loader = async () => {
+  const podcasts = await getPodcasts();
+  return json({ podcasts });
+};
+
+const Podcasts = () => {
+  const { podcasts } = useLoaderData<typeof loader>();
+  const featuredPodcasts = podcasts.filter((podcast) => podcast.featured);
```

Comment:
max 3 podcasts også?
```suggestion
  const featuredPodcasts = podcasts.filter((podcast) => podcast.featured).slice(0,3);
```